AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/CampaignController.cs

```diff
@@ -170,6 +170,43 @@ public IActionResult Create()
 
         [HttpPost]
         [ValidateAntiForgeryToken]
+        public async Task<JsonResult> DeleteCampaignImage(int campaignId)
+        {
+            if (campaignId == default(int))
+            {
+                return Json(new { status = "CampaignIdNotProvided" });
+            }
+
+            var campaign = await _mediator.SendAsync(new CampaignSummaryQuery { CampaignId = campaignId });
+
+            if (campaign == null)
+            {
+                return Json(new { status = "NotFound" });
+            }
+
+            if (!User.IsOrganizationAdmin(campaign.OrganizationId))
+            {
+                return Json(new { status = "Unauthorized" });
+            }
+
+            if (campaign.EndDate < campaign.StartDate)
+            {
+                return Json(new { status = "DateInvalid", message = "The end date must fall on or after the start date." });
+            }
+
+            if (campaign.ImageUrl != null)
+            {
+                await _imageService.DeleteImageAsync(campaign.ImageUrl);

```

Comment:
We are actually wanting to move away from "doing things" in the controller, but in this case you're only using the `IImageService` and not adding it to the controller. I think this is fine as-is, but we'll likely want to add an issue to refactor the service out of the controller (and towards the MediatR command/handler pattern we've adopted).